spec/helpers/expenses_helper_spec.rb

```diff
@@ -0,0 +1,15 @@
+require 'rails_helper'
+
+# Specs in this file have access to a helper object that includes
+# the ExpensesHelper. For example:
+#
+# describe ExpensesHelper do
+#   describe "string concat" do
+#     it "concats two strings with spaces" do
+#       expect(helper.concat_strings("this","that")).to eq("this that")
+#     end
+#   end
+# end
```

Comment:
- [ ] Kindly, implement the pending tests or you can remove the commented ones to keep the running tests only.

![image](https://user-images.githubusercontent.com/55635977/159103023-baad222b-a6d2-4090-a881-7bbe758c1ff8.png)

Reply:
This has been fixed